documentation/writing_tasks.md

```diff
@@ -421,6 +421,26 @@ end
 puts result.to_json
 ```
 
+### Returning sensitive data
+
+To return secrets from a task, use the `_sensitive` key in the output. Bolt
+will treat the result as sensitive and won't allow it to be printed to the
+console or log.
+
+```ruby
+#!/opt/puppetlabs/puppet/bin/ruby
+
+require 'json'
+
+user_name = 'someone'
+# Generate a 10 letter password
+user_password = [*'a'..'z'].sample(10).join
+
+result = { user: user_name, _sensitive: { password: user_password } }
```

Comment:
Should we also document a plan that unwraps and uses a sensitive value?

Reply:
We do have a section on "Sensitive parameters" which shows unwrapping a `Sensitive` instance in another context and there's a note on the `sensitive()` function of the `Result` datatype that says to use `unwrap()`. Pulling sensitive data off a `Result` seems to be at a finer level of detail than the rest of our writing plans documentation.